Tests/SpelledPitchTests/PitchSpellerTests.swift

```diff
@@ -13,12 +13,12 @@ import Pitch
 class PitchSpellerTests: XCTestCase {
```

Comment:
Can we add a `testSpellEightOverCNatural()` which verifies that we get an `(.a, .flat)` result?

Reply:
Yes, that's definitely the plan! I think there will be a bit more organizing needed before we can get a green light on that, but I can always throw in a failing test for now of course / cheat for now!

Reply:
How about: let's merge this without adding the test. But then, add a PR where the first commit is such a failing test. Then we can work to green-light it.